lib/manageiq/providers/ibm_cloud/cloud_tools/common.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+dir_name = File.basename(__FILE__).split('.')[0]
```

Comment:
Not for this PR, since we are close to merge but this is already provided by Ruby with the `__dir__` method

Reply:
No it isn't. This is the name of the file minus the .rb. dir_name is a reference to the target directory for the relative import. So in this case common.rb will look for ruby files in the 'common' directory and import them relatively.

Reply:
ohhh...yeah you're right...in that case you can also use `File.basename(__FILE__, ".*")`